3.3.0 missing multiple definitions?

I updated from 3.2.0 to 3.3.0 in my 2022.3.56 Unity project and I get 27 errors:

```
Library/PackageCache/com.timcassell.protopromise@3.3.0/Core/Timers/Internal/SystemTimerFactoryInternal.cs(40,65): error CS0234: The type or namespace name 'HandleablePromiseBase' does not exist in the namespace 'Internal' (are you missing an assembly reference?)

Library/PackageCache/com.timcassell.protopromise@3.3.0/Core/Timers/Internal/SystemTimerFactoryInternal.cs(157,61): error CS0234: The type or namespace name 'PromiseRefBase' does not exist in the namespace 'Internal' (are you missing an assembly reference?)

Library/PackageCache/com.timcassell.protopromise@3.3.0/Core/Timers/Internal/SystemTimerFactoryInternal.cs(157,104): error CS0234: The type or namespace name 'VoidResult' does not exist in the namespace 'Internal' (are you missing an assembly reference?)

Library/PackageCache/com.timcassell.protopromise@3.3.0/Core/Timers/Internal/TimeProviderTimerFactoryInternal.cs(25,40): error CS0234: The type or namespace name 'LocalObjectPool<>' does not exist in the namespace 'Internal' (are you missing an assembly reference?)

Library/PackageCache/com.timcassell.protopromise@3.3.0/Core/Timers/Internal/TimeProviderTimerFactoryInternal.cs(50,71): error CS0234: The type or namespace name 'HandleablePromiseBase' does not exist in the namespace 'Internal' (are you missing an assembly reference?)

Library/PackageCache/com.timcassell.protopromise@3.3.0/Core/Timers/Internal/TimeProviderTimerFactoryInternal.cs(169,61): error CS0234: The type or namespace name 'PromiseRefBase' does not exist in the namespace 'Internal' (are you missing an assembly reference?)

Library/PackageCache/com.timcassell.protopromise@3.3.0/Core/Timers/Internal/TimeProviderTimerFactoryInternal.cs(169,104): error CS0234: The type or namespace name 'VoidResult' does not exist in the namespace 'Internal' (are you missing an assembly reference?)

Library/PackageCache/com.timcassell.protopromise@3.3.0/Core/Timers/Internal/SystemTimerFactoryInternal.cs(69,25): error CS0234: The type or namespace name 'InlineOption' does not exist in the namespace 'Internal' (are you missing an assembly reference?)

Library/PackageCache/com.timcassell.protopromise@3.3.0/Core/Timers/Internal/SystemTimerFactoryInternal.cs(78,25): error CS0234: The type or namespace name 'InlineOption' does not exist in the namespace 'Internal' (are you missing an assembly reference?)

Library/PackageCache/com.timcassell.protopromise@3.3.0/Core/Timers/Internal/SystemTimerFactoryInternal.cs(166,29): error CS0234: The type or namespace name 'InlineOption' does not exist in the namespace 'Internal' (are you missing an assembly reference?)

Library/PackageCache/com.timcassell.protopromise@3.3.0/Core/Timers/Internal/SystemTimerFactoryInternal.cs(175,29): error CS0234: The type or namespace name 'InlineOption' does not exist in the namespace 'Internal' (are you missing an assembly reference?)

Library/PackageCache/com.timcassell.protopromise@3.3.0/Core/Timers/Internal/SystemTimerFactoryInternal.cs(200,29): error CS0234: The type or namespace name 'InlineOption' does not exist in the namespace 'Internal' (are you missing an assembly reference?)

Library/PackageCache/com.timcassell.protopromise@3.3.0/Core/Timers/Internal/SystemTimerFactoryInternal.cs(208,29): error CS0234: The type or namespace name 'InlineOption' does not exist in the namespace 'Internal' (are you missing an assembly reference?)

Library/PackageCache/com.timcassell.protopromise@3.3.0/Core/Timers/Internal/SystemTimerFactoryInternal.cs(242,29): error CS0234: The type or namespace name 'InlineOption' does not exist in the namespace 'Internal' (are you missing an assembly reference?)

Library/PackageCache/com.timcassell.protopromise@3.3.0/Core/Timers/Internal/SystemTimerFactoryInternal.cs(262,29): error CS0234: The type or namespace name 'InlineOption' does not exist in the namespace 'Internal' (are you missing an assembly reference?)

Library/PackageCache/com.timcassell.protopromise@3.3.0/Core/Timers/Internal/SystemTimerFactoryInternal.cs(269,29): error CS0234: The type or namespace name 'InlineOption' does not exist in the namespace 'Internal' (are you missing an assembly reference?)

Library/PackageCache/com.timcassell.protopromise@3.3.0/Core/Timers/Internal/SystemTimerFactoryInternal.cs(278,29): error CS0234: The type or namespace name 'InlineOption' does not exist in the namespace 'Internal' (are you missing an assembly reference?)

Library/PackageCache/com.timcassell.protopromise@3.3.0/Core/Timers/Internal/TimeProviderTimerFactoryInternal.cs(81,25): error CS0234: The type or namespace name 'InlineOption' does not exist in the namespace 'Internal' (are you missing an assembly reference?)

Library/PackageCache/com.timcassell.protopromise@3.3.0/Core/Timers/Internal/TimeProviderTimerFactoryInternal.cs(90,25): error CS0234: The type or namespace name 'InlineOption' does not exist in the namespace 'Internal' (are you missing an assembly reference?)

Library/PackageCache/com.timcassell.protopromise@3.3.0/Core/Timers/Internal/TimeProviderTimerFactoryInternal.cs(179,29): error CS0234: The type or namespace name 'InlineOption' does not exist in the namespace 'Internal' (are you missing an assembly reference?)

Library/PackageCache/com.timcassell.protopromise@3.3.0/Core/Timers/Internal/TimeProviderTimerFactoryInternal.cs(188,29): error CS0234: The type or namespace name 'InlineOption' does not exist in the namespace 'Internal' (are you missing an assembly reference?)

Library/PackageCache/com.timcassell.protopromise@3.3.0/Core/Timers/Internal/TimeProviderTimerFactoryInternal.cs(214,29): error CS0234: The type or namespace name 'InlineOption' does not exist in the namespace 'Internal' (are you missing an assembly reference?)

Library/PackageCache/com.timcassell.protopromise@3.3.0/Core/Timers/Internal/TimeProviderTimerFactoryInternal.cs(222,29): error CS0234: The type or namespace name 'InlineOption' does not exist in the namespace 'Internal' (are you missing an assembly reference?)

Library/PackageCache/com.timcassell.protopromise@3.3.0/Core/Timers/Internal/TimeProviderTimerFactoryInternal.cs(256,29): error CS0234: The type or namespace name 'InlineOption' does not exist in the namespace 'Internal' (are you missing an assembly reference?)

Library/PackageCache/com.timcassell.protopromise@3.3.0/Core/Timers/Internal/TimeProviderTimerFactoryInternal.cs(276,29): error CS0234: The type or namespace name 'InlineOption' does not exist in the namespace 'Internal' (are you missing an assembly reference?)

Library/PackageCache/com.timcassell.protopromise@3.3.0/Core/Timers/Internal/TimeProviderTimerFactoryInternal.cs(283,29): error CS0234: The type or namespace name 'InlineOption' does not exist in the namespace 'Internal' (are you missing an assembly reference?)

Library/PackageCache/com.timcassell.protopromise@3.3.0/Core/Timers/Internal/TimeProviderTimerFactoryInternal.cs(292,29): error CS0234: The type or namespace name 'InlineOption' does not exist in the namespace 'Internal' (are you missing an assembly reference?)
```

![Image](https://github.com/user-attachments/assets/e570d698-b7ae-4524-a239-a0cd1d156f2e)

--- a/Package/Core/ProtoPromise.asmdef
+++ b/Package/Core/ProtoPromise.asmdef
@@ -5,9 +5,8 @@
     "includePlatforms": [],
     "excludePlatforms": [],
     "allowUnsafeCode": true,
-    "overrideReferences": false,
+    "overrideReferences": true,
     "precompiledReferences": [],
     "autoReferenced": true,
-    "defineConstraints": [],
-    "versionDefines": []
+    "defineConstraints": []
 }

--- a/Package/UnityHelpers/2018.3/ProtoPromise.UnityHelpers.2018.3.asmdef
+++ b/Package/UnityHelpers/2018.3/ProtoPromise.UnityHelpers.2018.3.asmdef
@@ -7,7 +7,7 @@
     "includePlatforms": [],
     "excludePlatforms": [],
     "allowUnsafeCode": true,
-    "overrideReferences": false,
+    "overrideReferences": true,
     "precompiledReferences": [],
     "autoReferenced": true,
     "defineConstraints": []

--- a/Package/UnityHelpers/2023.1/ProtoPromise.UnityHelpers.2023.1.asmdef
+++ b/Package/UnityHelpers/2023.1/ProtoPromise.UnityHelpers.2023.1.asmdef
@@ -1,16 +1,14 @@
 {
     "name": "ProtoPromise.UnityHelpers.2023.1",
-    "rootNamespace": "",
     "references": [
         "ProtoPromise"
     ],
+    "optionalUnityReferences": [],
     "includePlatforms": [],
     "excludePlatforms": [],
     "allowUnsafeCode": false,
-    "overrideReferences": false,
+    "overrideReferences": true,
     "precompiledReferences": [],
     "autoReferenced": true,
-    "defineConstraints": [],
-    "versionDefines": [],
-    "noEngineReferences": false
+    "defineConstraints": []
 }

